MachObfuscatorTests/MachTests/Mach+Replacing_cstrings_Tests.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Mach+Replacing_cstrings_Tests.swift
+//  MachObfuscatorTests
+//
+
+import XCTest
+
+class Mach_Replacing_cstring_Tests: XCTestCase {
+    
+    var sut: Image!
+    var originalCstrings: Set<String>!
+    //These propeties require that cstring reflects current state of image data (i.e. is not lazily computed and later not updated)
+    var currentCstrings: Set<String> { return Set(sut.machs[0].cstrings) }
+    var newCstrings: Set<String> { return currentCstrings.subtracting(originalCstrings) }
+    var removedCstrings: Set<String> { return originalCstrings.subtracting(currentCstrings) }
+ 
+    override func setUp() {
+        super.setUp()
+        
+        sut = try! Image.load(url: URL.machoMacExecutable)
+        originalCstrings = currentCstrings
+    }
+    
+    override func tearDown() {
+        sut = nil
+        super.tearDown()
+    }
+    
+    private func assertNothingChanged() {
```

Comment:
What do you think about improving this function by passing the exact line numbers.
```
private func assertNothingChanged(file: String = #file, line: Int = #line) {
    XCTAssert(newCstrings.isEmpty, file: file, line: line)
    XCTAssert(removedCstrings.isEmpty, file: file, line: line)
```

Reply:
ok, changed